prewl/endpoints/gpt2.py

```diff
@@ -0,0 +1,21 @@
+from prewl.endpoints.base import Endpoint
+
+class GPT2(Endpoint):
+    
+    def call(self, prompt):
+        import os
+        os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
+        import tensorflow as tf
+        tf.compat.v1.logging.set_verbosity(tf.compat.v1.logging.ERROR)
+        from transformers import TFGPT2LMHeadModel, GPT2Tokenizer
+        tokenizer = GPT2Tokenizer.from_pretrained("gpt2")
+        model = TFGPT2LMHeadModel.from_pretrained("gpt2", pad_token_id=tokenizer.eos_token_id)
+
+        # encode context the generation is conditioned on
+        input_ids = tokenizer.encode(prompt, return_tensors='tf')
+
+        # generate text until the output length (which includes the context length) reaches 50
+        greedy_output = model.generate(input_ids, max_length=100)
```

Comment:
The `max_length` should come from the prewl settings.

Are there any other settings we might use here or in the next line?